packages/earn-controller/src/EarnController.ts

```diff
@@ -95,23 +126,49 @@ const earnControllerMetadata: StateMetadata<EarnControllerState> = {
 // === State Types ===
 export type EarnControllerState = {
   pooled_staking: PooledStakingState;
-  stablecoin_lending?: StablecoinLendingState;
+  lending: LendingState;
```

Comment:
These changes to `EarnControllerState` (removing `stablecoin_lending`, adding `lending`) look like breaking changes to me, what do you think about documenting them in the changelog?